examples/cube/main.rs

```diff
@@ -173,15 +173,17 @@ impl framework::Example for Example {
             height: size,
             depth: 1,
         };
-        let texture = device.create_texture(&wgpu::TextureDescriptor {
-            label: None,
-            size: texture_extent,
-            mip_level_count: 1,
-            sample_count: 1,
-            dimension: wgpu::TextureDimension::D2,
-            format: wgpu::TextureFormat::Rgba8UnormSrgb,
-            usage: wgpu::TextureUsage::SAMPLED | wgpu::TextureUsage::COPY_DST,
-        });
+        let texture = device.create_texture(
+            wgpu::TextureDescriptor::new(
+                texture_extent,
+                wgpu::TextureDimension::D2,
+                wgpu::TextureFormat::Rgba8UnormSrgb,
+                wgpu::TextureUsage::SAMPLED | wgpu::TextureUsage::COPY_DST,
+            )
+            .label("label")
```

Comment:
Is rustfmt really formatting it this way?

Reply:
Yeah. Would you've formatted it differently?

Reply:
Typically if you chain calls, they get aligned with indentation relative to the init line, i.e.
```rust
something
  .do_something()
  .and_something()
```

In our case, it looks like there is no indentation, so I'm surprised

Reply:
that's the expected behavior inside parenthesized expressions I think